docs/source/quickstart.md

```diff
@@ -14,235 +18,193 @@ pip install mammoth-nlp
+Set the locations to the directory in which you want to work on this project, and the path to the mammoth git repo.
 
-### Step 2: Configurations
+```bash
+export PROJECT_DIR="/path/to/work/dir"
+export MAMMOTH_DIR="/path/to/mammoth"
+```
 
-Mamooth uses configurations to build a new transformer model and configure your training settings, such as which modules are trained with the data from which languages.
+### Step 2: Activate your virtual env
 
-Below are a few examples of training configurations that will work for you out-of-box in a one-node, two-GPU environment.
+```bash
+source ~/venvs/mammoth/bin/activate
+```
 
-<details>
-<summary>Task-specific encoders and decoders</summary>
-
-In this example, we create a model with encoders and decoders **unshared** for the specified languages. This is defined by `enc_sharing_group` and `enc_sharing_group`.
-Note that the configs expect you have access to 2 GPUs.
-
-```yaml
-# TRAINING CONFIG
-world_size: 2
-gpu_ranks: [0, 1]
-
-batch_type: tokens
-batch_size: 4096
-
-# INPUT/OUTPUT VOCABULARY CONFIG
+### Step 3: Copy the config template from the Mammoth repo
 
-src_vocab:
-  bg: vocab/opusTC.mul.vocab.onmt
-  cs: vocab/opusTC.mul.vocab.onmt
-  en: vocab/opusTC.mul.vocab.onmt
-tgt_vocab:
-  cs: vocab/opusTC.mul.vocab.onmt
-  en: vocab/opusTC.mul.vocab.onmt
+```bash
+cd $PROJECT_DIR
+mkdir config
+cp -i ${MAMMOTH_DIR}/examples/synthdata.template.yaml config/synthdata.template.yaml
+```
 
-# MODEL CONFIG
+### Step 4: Generate synthetic data
 
-model_dim: 512
+(this might take about 5 min)
 
-tasks:
-  train_bg-en:
-    src_tgt: bg-en
-    enc_sharing_group: [bg]
-    dec_sharing_group: [en]
-    node_gpu: "0:0"
-    path_src: europarl_data/encoded/train.bg-en.bg.sp
-    path_tgt: europarl_data/encoded/train.bg-en.en.sp
-  train_cs-en:
-    src_tgt: cs-en
-    enc_sharing_group: [cs]
-    dec_sharing_group: [en]
-    node_gpu: "0:1"
-    path_src: europarl_data/encoded/train.cs-en.cs.sp
-    path_tgt: europarl_data/encoded/train.cs-en.en.sp
-  train_en-cs:
-    src_tgt: en-cs
-    enc_sharing_group: [en]
-    dec_sharing_group: [cs]
-    node_gpu: "0:1"
-    path_src: europarl_data/encoded/train.cs-en.en.sp
-    path_tgt: europarl_data/encoded/train.cs-en.cs.sp
-
-enc_layers: [6]
-dec_layers: [6]
-```
-</details>
-
-
-<details>
-<summary>Arbitrarily shared layers in encoders and task-specific decoders</summary>
-
-The training and vocab config is the same as in the previous example.
-
-```yaml
-# TRAINING CONFIG
-world_size: 2
-gpu_ranks: [0, 1]
-
-batch_type: tokens
-batch_size: 4096
-
-# INPUT/OUTPUT VOCABULARY CONFIG
+```bash
+python ${MAMMOTH_DIR}/tools/generate_synth_data.py \
```

Comment:
is this part of standard tools? if yes, why do require git cloning ; if no, can't we just suggesting wgetting?

Reply:
Fixed in #86